### PR TITLE
Update README w/ install instruction for R v3.5.0

### PR DESCRIPTION
**Pull request by jeffeaton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ drat:::add("mrc-ide")
 install.packages("didehpc")
 ```
 
-or try the slightly experimental:
+For installation on R v3.5.0, use the slightly experimental:
```
Comment (richfitz) [inline README.md]: Why do you have this on 3.5.0? I think you're being mislead by R's terrible installation failure message perhaps ("not available for R version x.y.z") - both methods should work fine for all current R versions I believe

**Comment by jeffeaton** [comment]
Yeah, just uninstalled and tested

drat:::add("mrc-ide")
install.packages("didehpc”)

It worked fine now. Not sure why it didn’t work last week…

**Comment by richfitz** [comment]
Mind if I close this then?

**Comment by jeffeaton** [comment]
Please do

**Pull request closed without merging**